src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROWTH_FACTOR = 1.5;
+    private static final int DEFAULT_INDEX = 0;
+    private static final String INDEX_OUT_OF_BOUNDS_EXCEPTION_MESSAGE
+            = "Index out of bounds: ";
+    private static final String NO_SUCH_ELEMENT_EXCEPTION_MESSAGE = "Element not found in the list";
+    private T[] elementData = (T[]) new Object[DEFAULT_CAPACITY];
+    private int size;
+
     @Override
     public void add(T value) {
-
+        growIfArrayFull();
+        elementData[size] = value;
+        size++;
```

Comment:
you can do that in 1 row

Reply:
I'm not sure, but perhaps on QnA, the mentor told us better to use size++ under the line, need to reask